anta/tests/flow_tracking.py

```diff
@@ -99,94 +88,51 @@ class VerifyHardwareFlowTrackerStatus(AntaTest):
-            hardware_tracker_name = command.params.name
-            record_export = tracker_input.record_export.model_dump() if tracker_input.record_export else None
-            exporters = [exporter.model_dump() for exporter in tracker_input.exporters] if tracker_input.exporters else None
-            command_output = command.json_output
 
-            # Check if hardware flow tracking is configured
-            if not command_output.get("running"):
-                self.result.is_failure("Hardware flow tracking is not running.")
-                return
+        command_output = self.instance_commands[0].json_output
+        # Check if hardware flow tracking is configured
+        if not command_output.get("running"):
+            self.result.is_failure("Hardware flow tracking is not running.")
+            return
 
+        for tracker in self.inputs.trackers:
             # Check if the input hardware tracker is configured
-            tracker_info = command_output["trackers"].get(hardware_tracker_name)
-            if not tracker_info:
-                self.result.is_failure(f"Hardware flow tracker `{hardware_tracker_name}` is not configured.")
+            if not (tracker_info := get_value(command_output["trackers"], f"{tracker.name}")):
+                self.result.is_failure(f"{tracker} - Not found")
                 continue
 
             # Check if the input hardware tracker is active
             if not tracker_info.get("active"):
-                self.result.is_failure(f"Hardware flow tracker `{hardware_tracker_name}` is not active.")
+                self.result.is_failure(f"{tracker} - Disabled")
                 continue
 
             # Check the input hardware tracker timeouts
-            failure_msg = ""
-            if record_export:
-                record_export_failure = validate_record_export(record_export, tracker_info)
-                if record_export_failure:
-                    failure_msg += record_export_failure
-
-            # Check the input hardware tracker exporters' configuration
-            if exporters:
-                exporters_failure = validate_exporters(exporters, tracker_info)
-                if exporters_failure:
-                    failure_msg += exporters_failure
-
-            if failure_msg:
-                self.result.is_failure(f"{hardware_tracker_name}: {failure_msg}\n")
+            if tracker.record_export:
+                inactive_interval = tracker.record_export.on_inactive_timeout
+                on_interval = tracker.record_export.on_interval
+                act_inactive = tracker_info.get("inactiveTimeout")
+                act_interval = tracker_info.get("activeInterval")
+                if not all([inactive_interval == act_inactive, on_interval == act_interval]):
+                    self.result.is_failure(
+                        f"{tracker} {tracker.record_export} - Incorrect durations - InActive Timeout: {act_inactive} OnActive Interval: {act_interval}"
```

Comment:
missing the actual / expected so we don't know what value we are getting from this lg message
```suggestion
                        f"{tracker} {tracker.record_export} - Incorrect durations - InActive Timeout: {act_inactive}, OnActive Interval: {act_interval}"
```

Reply:
Updated